src/main/scala/io/github/pauljamescleary/petstore/Server.scala

```diff
@@ -13,6 +13,7 @@ import fs2.{Stream, StreamApp}
 import org.http4s.server.blaze.BlazeBuilder
 
 object Server extends StreamApp[IO] {
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
odd, i see where it was added by http4s here: https://github.com/http4s/http4s/commit/e5ca100a1a98466083b40aa25edb5cfb497b1186#diff-82f67a5c7812fb25ab96e372077ad6e9

because fs2 changed somehow.

Reply:
yes, it didnt compile without it